Bowling.java

```diff
@@ -117,6 +113,18 @@ public Integer getScore() {
 		return totalPoints;
 	}
 
+	public Boolean isGameOver() {
+		currentFrame = getCurrentFrame();
+		secondToLastFrame = getFrameBefore(currentFrame);
+		if (secondToLastFrame.isPresent()) {
+			thirdToLastFrame = getFrameBefore(secondToLastFrame.get());
+		}
+		
+		return allFrames.size() == 11 && 
+			   !secondToLastFrame.get().isStrike() && 
+			   !secondToLastFrame.get().isSpare();
```

Comment:
Hi-- Its going to take me a beat to get my brain warmed up to this again. I hope it won't take me long!

To answer your last question, nope that wasn't the simplest code I could have written, but you already knew that :). I jumped ahead and made an assumption. I will make a change to the `isGameOver` method. 

Is there anything else you were hoping I would address before merging the pull request?